headapps/MvpSite/MvpSite.Rendering/Views/Shared/Components/SitecoreComponent/_clFeatureBlock.cshtml

```diff
@@ -1,13 +1,15 @@
 ﻿@model MvpSite.Rendering.Models.ContentListItem
 
-<div class="col-12 col-lg-6">
-    <div class="feature-panel-text with-background no-number">
-        <h2 asp-for="@Model.Fields!.ItemTitle"></h2>
+<div class="feature-panel-card">
+    <div class="feature-panel-text  no-number ">
```

Comment:
double space and trailing space in class? Does this do anything specific? I noticed this a few times